packages/cli/src/util/client.ts

```diff
@@ -78,34 +85,30 @@ export default class Client extends EventEmitter {
       }
 
       _url = `${apiUrl}${parsedUrl.pathname}?${query}`;
-
-      delete opts.useCurrentTeam;
-      delete opts.accountId;
     }
 
-    if (opts.json !== false && opts.body && typeof opts.body === 'object') {
```

Comment:
Do we no longer support `opts.json: false`?

Reply:
It's supported for making the return value be `res` instead of the parsed response body.

But for the request body it's not necessary since the `isJSONObject()` check is in place. The [two](https://github.com/vercel/vercel/blob/bf3a1fc832f0d131772f1728950c20c9ce2b6275/packages/cli/src/util/dns/import-zonefile.ts#L31) [places](https://github.com/vercel/vercel/blob/930063638c4ddeb5ed475269a5133d39e040a182/packages/cli/src/util/events.ts) where `json: false` is currently being used are not passing in an object body anyways.